alibi_detect/base.py

```diff
@@ -155,26 +152,16 @@ def from_config(cls, config: dict):
         return detector
 
     def _set_config(self, inputs):  # TODO - move to BaseDetector once config save/load implemented for non-drift
-        """
```

Comment:
Should we keep some part of this docstring?

Reply:
Good point, overly aggressive pruning on my part. Will add back in!